### PR TITLE
SAK-49442 gradebook. Added tooltips to column headers

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-49442

### CODE REVIEW

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java [hunk 1]
```diff
@@ -63,6 +64,8 @@ public class GbGradebookData {
 	private static final String SAK_PROP_SHOW_COURSE_GRADE_STUDENT = "gradebookng.showDisplayCourseGradeToStudent";
 	private static final Boolean SAK_PROP_SHOW_COURSE_GRADE_STUDENT_DEFAULT = true;
 
+    private static ResourceLoader i18n = new ResourceLoader("gradebookng");
```
Comment (jesusmmp) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java:67]: Spacing

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java [hunk 2]
```diff
@@ -211,13 +215,15 @@ public DataSet(final List<StudentDefinition> students,
...
 				final String serializedGrades,
+				final ResourceLoader i18n,
 				final Map<String, Object> settings,
 				final Long courseGradeId,
 				final Long gradebookId) {
 			this.students = students;
 			this.columns = columns;
 			this.courseGrades = courseGrades;
 			this.serializedGrades = serializedGrades;
+            this.i18n = i18n;
```
Comment (jesusmmp) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java:226]: Spacing

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java [hunk 3]
```diff
@@ -286,6 +292,7 @@ public String toScript() {
 				GbGradebookData.this.columns,
 				courseGrades(),
 				serializeGrades(grades),
+                GbGradebookData.this.i18n,
```
Comment (jesusmmp) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java:295]: Spacing

**Comment by adrianfish** [comment]
Thanks for the review @jesusmmp :)

**Review by jesusmmp — APPROVED** [review]

**Pull request merged**